shared/types/api/node.go

```diff
@@ -403,7 +403,7 @@ type CreateVacantMinipoolResponse struct {
 type CanNodeSendResponse struct {
 	Status              string             `json:"status"`
 	Error               string             `json:"error"`
-	Balance             *big.Int           `json:"balance"`
+	Balance             float64            `json:"balance"`
```

Comment:
We should really switch to something like this at some point:
<https://pkg.go.dev/github.com/shopspring/decimal>